app/views/layouts/application.html.erb

```diff
@@ -1,12 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
`Ruby on Rails Tutorial Sample App`
chú tìm hiểu về `I18n` và Sử dụng I18n cho toàn bộ text của toàn bộ file html nhé